Sparkle/SUGuidedPackageInstaller.m

```diff
@@ -55,11 +55,11 @@ + (AuthorizationRef)authorizationForExecutable:(NSString*)executablePath
 		const char* executableFileSystemRepresentation = [executablePath fileSystemRepresentation];
 		
 		// Prepare a right allowing script to execute with privileges
-		AuthorizationItem right;
-		memset(&right,0,sizeof(right));
-		right.name = kAuthorizationRightExecute;
-		right.value = (void*) executableFileSystemRepresentation;
-		right.valueLength = strlen(executableFileSystemRepresentation);
+        AuthorizationItem right = {
```

Comment:
This is causing the Travis build failure I think. Just revert this part of the PR.